README.md

```diff
@@ -11,3 +9,59 @@ Make sure that you have the compatible versions of Visual Studio with .NET MAUI
 ## How to run this application?
 
 To run this application, you need to first clone the getting-started-with-dotnet-maui-scheduler repository and then open it in Visual Studio 2022. Now, simply build and run your project to view the output.
+
+## Features and Benefits
+
+### Recurring events
+Easily configure recurring events on a daily, weekly, monthly, or yearly basis. Skip or change the occurrence of a recurring appointment.
+
+### Agenda view
+Display the appointments (normal, all day, recurring, and span) in a chronological order, grouped by date.
+
+### Resource view
+Group appointments based on the resources associated with them in the timeline day, timeline week, timeline workweek, and timeline month views.
+
+### Right to left (RTL)
+The .NET MAUI Scheduler supports displaying the Scheduler component from right to left for users working with RTL languages like Hebrew and Arabic.
+
+## Related Links
+
+[Learn More about .NET MAUI Scheduler](https://www.syncfusion.com/maui-controls/maui-scheduler?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace)<br/><br/>
+
+[Download Free Trial](https://www.syncfusion.com/downloads/maui/confirm?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace) <br/><br/>
```

Comment:
Use the below link for download free trial:
https://www.syncfusion.com/downloads/maui

Reply:
Updated